plugins/ccgx/ccgx.quirk

```diff
@@ -112,3 +112,9 @@ CcgxDmcTriggerCode = 0x01
 CcgxImageKind = dmc-composite
 InstallDuration = 898
 RemoveDelay = 732000
+
+# Intel Gatkex Creek Goshen Ridge EVB
+[USB\VID_04B4&PID_5220]
```

Comment:
Are we sure that this exact PID is unique to Gatkex creek? If not - it should go into a noinst ccgx quirk.

Reply:
@chlee75 please